butter-cms.json

```diff
@@ -53,13 +53,13 @@
     "SDK [TypeScript]": "No",
     "Web Hooks": "Yes",
     "Eventbus": "No",
-    "Bulk Import": "No",
+    "Bulk Import": "Yes",
```

Comment:
Theoretically, you can do this programmatically through their **Write API**.

Reply:
Bulk import is a dedicated feature. I would otherwise not have listed it. Some systems have a dedicated API endpoint that for example accepts ND-Json encoded data to be processed at once.